docs/superpowers/plans/2026-05-01-postgresql-queue.md

```
@@ -0,0 +1,1830 @@
+		cfg.Driver, cfg.MaxIdleConns, cfg.MaxOpenConns)
+}
+```
+
+- [ ] **Step 4: Run tests — expect PASS**
+
+```bash
+go test ./internal/infra/database/... -v
+```
+
+Expected:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)